sdk/src/types/block/slot/index.rs

```diff
@@ -42,20 +41,16 @@ use crate::types::block::Error;
     packable::Packable,
 )]
 #[repr(transparent)]
-pub struct SlotIndex(u64);
+pub struct SlotIndex(pub u64);
 
 impl SlotIndex {
-    /// Creates a new [`SlotIndex`].
-    pub fn new(index: u64) -> Self {
-        Self::from(index)
+    /// Gets the [`EpochIndex`] of this slot.
+    pub fn to_epoch_index(self, slots_per_epoch_exponent: u8) -> EpochIndex {
+        EpochIndex::from_slot_index(self, slots_per_epoch_exponent)
     }
```

Comment:
```suggestion
    /// Converts the slot index into the [`EpochIndex`] of this slot.
    pub fn to_epoch_index(self, slots_per_epoch_exponent: u8) -> EpochIndex {
        EpochIndex::from_slot_index(self, slots_per_epoch_exponent)
    }
```

Reply:
I don't think that's better

Reply:
not from me though :joy: ...it's from what's already there, but okay